ehitchhiking-ui/src/app/pages/registration/registration.component.html

```diff
@@ -0,0 +1,33 @@
+<div class = "page">
+<div id="login-form">
+    <div class = "exadel-login">
+      <img src="../../../assets/images/FullColor_Word-1.png" alt="exadel-logo"/>
+    </div>
+
+  <fieldset class = "user-info">
+    <form class = "form-img">
+      <form (submit)="onSubmit($event)">
+        <div class = "location-icon">
```

Comment:
what is location-icon ?

Reply:
in this case, the location of the image changes relative to input